include/sway/criteria.h

```diff
@@ -6,6 +6,8 @@
 #include "list.h"
 #include "tree/view.h"
 
+struct sway_container;
```

Comment:
I don't think this is needed. `struct sway_container` is not referenced in the header and  `sway/criteria.c` already includes `sway/tree/container.h`.